src/V3EmitCMake.cpp

```diff
@@ -26,6 +26,8 @@
 
 VL_DEFINE_DEBUG_FUNCTIONS;
 
+// TODO: this file is completely uncovered by tests
+
```

Comment:
The test suite has 0 coverage for this file. I suspec that's because the cmake builds now go via `--make json`, so this an likely be removed, but what should `--make cmake` do?

Reply:
We certainly use this. It's probably because we aren't enabling coverage for cmake, right now we only edit the .mk.in which cmake doesn't use.

Reply:
BTW. with `configure --enabel-dev-gcov`, you will get code coverage for all invocation of verilator_bin*, even if via CMake. The only thing that would have coverage is if you build verilator_bin* with CMake, which we don't do.

Reply:
Ah, yes, guess the only path is through the json output now.  Ok, feel free to make a pre or post-pull to kill that whole file.